plugins/modules/zpool.py

```diff
@@ -0,0 +1,588 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2025, Tom Hesse <contact@tomhesse.xyz>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+module: zpool
+short_description: Manage ZFS zpools
+description:
+  - Create, destroy, and modify ZFS zpools and their vdev layouts, pool properties, and filesystem properties.
+extends_documentation_fragment:
+  - community.general.attributes
+attributes:
+  check_mode:
+    support: partial
+    details:
+      - In check_mode, any zpool subcommand that supports the dry-run flag (`-n`) will be run with `-n` and its
+        simulated output included in the module's diff results.
+  diff_mode:
+    support: full
+author:
+  - Tom Hesse (@tomhesse)
+options:
+  name:
+    description:
+      - Name of the zpool to manage.
+    required: true
+    type: str
+  state:
+    description:
+      - Whether the pool should exist.
+    choices: [ present, absent ]
+    default: present
+    type: str
+  disable_new_features:
+    description:
+      - If true, disable new ZFS feature flags when creating.
+    type: bool
+    default: false
+  force:
+    description:
+      - If true, force operations (e.g. overwrite existing devices).
```

Comment:
```suggestion
      - If true, force operations (for example overwrite existing devices).
```
https://docs.ansible.com/ansible/devel/dev_guide/style_guide/#avoid-using-latin-phrases